package.json

```diff
@@ -32,20 +32,18 @@
     },
     "homepage": "https://github.com/BigRoomStudios/schwifty-migration#readme",
     "dependencies": {
-        "chalk": "^1.1.3",
         "handlebars": "^4.0.8",
-        "inquirer": "^2.0.0",
+        "hoek": "^5.0.3",
         "items": "^2.1.1",
         "joi": "^13.0.1",
         "jsondiffpatch": "^0.2.4",
-        "knex": "^0.13.0",
-        "toys": "^1.0.0"
+        "knex": "^0.13.0"
     },
     "peerDependencies": {
         "hapi": ">=10 <17"
     },
     "devDependencies": {
-        "bluebird": "^3.5.0",
+        "bluebird": "^3.5.1",
```

Comment:
If it's useful, you can also just get this off of a knex instance, `knex.Promise`.

Reply:
FYI grabbing promise off the Knex export (require('knex')) is deprecated -- grabbing it off an instance is fine though I think

Reply:
Can also grab Promise from objection